File: Software/MenuPrototype/Controls/SelectEventArgs.cs
namespace MenuPrototype
{
	partial class Program
	{
		////////////////////////////////////////////////////////////////////
		public class SelectEventArgs : EventArgs
		{
			string _selected;
			public SelectEventArgs(MenuItem source) : base(source) { }
			public SelectEventArgs(MenuItem source, string selected) : base(source) { _selected = selected; }
			public string getSelected() { return _selected; }
		}
	}
}

File: Software/Arduino code/OpenAstroTracker/lib/menu/controls/SelectEventArgs.cpp
#include <Arduino.h>

#include "SelectEventArgs.hpp"

SelectEventArgs::SelectEventArgs(MenuItem *source) : EventArgs(source)
{
}

SelectEventArgs::SelectEventArgs(MenuItem *source, String selected) : EventArgs(source)
{
	_selected = selected;
}

String SelectEventArgs::getSelected()
{
	return _selected;
}


